command/license.go

```diff
@@ -47,20 +47,27 @@ func (l *LicenseCommand) Run(args []string) int {
 func OutputLicenseReply(ui cli.Ui, resp *api.LicenseReply) int {
 	now := time.Now()
 	expired := resp.License.ExpirationTime.Before(now)
-	outputLicenseInfo(ui, resp.License, expired)
+	terminated := resp.License.TerminationTime.Before(now)
+	outputLicenseInfo(ui, resp.License, expired, terminated)
 	if expired {
 		return 1
 	}
 	return 0
 }
 
-func outputLicenseInfo(ui cli.Ui, lic *api.License, expired bool) {
+func outputLicenseInfo(ui cli.Ui, lic *api.License, expired, terminated bool) {
 	expStr := ""
 	if expired {
 		expStr = fmt.Sprintf("Expired At|%s", lic.ExpirationTime.String())
 	} else {
 		expStr = fmt.Sprintf("Expires At|%s", lic.ExpirationTime.String())
 	}
+	termStr := ""
+	if terminated {
+		termStr = fmt.Sprintf("Terminates At|%s", lic.TerminationTime.String())
+	} else {
+		termStr = fmt.Sprintf("Terminated At|%s", lic.TerminationTime.String())
```

Comment:
tenses should be swapped eh?

```suggestion
	if terminated {
		termStr = fmt.Sprintf("Terminated At|%s", lic.TerminationTime.String())
	} else {
		termStr = fmt.Sprintf("Terminates At|%s", lic.TerminationTime.String())
```

Reply:
Oops, good catch!